tests/test_util/test_util_display.py

```diff
@@ -2,8 +2,8 @@
 
 import pytest
 
-from sphinx.testing.util import strip_escseq
 from sphinx.util import logging
+from sphinx.util.console import strip_colors
```

Comment:
```suggestion
from sphinx.util.console import strip_colors as strip_escseq
```

(same reasoning as my previous comment)

Reply:
(same answer)